processing/src/main/java/io/druid/segment/DoubleDimensionMergerV9.java

```diff
@@ -57,6 +64,7 @@ public DoubleDimensionMergerV9(
     this.outDir = outDir;
     this.ioPeon = ioPeon;
     this.progress = progress;
+    this.nullRowsBitmap = indexSpec.getBitmapSerdeFactory().getBitmapFactory().makeEmptyMutableBitmap();
```

Comment:
Sometimes you are using the legacy static bitmap factory and sometimes you are using the one injected by Guice, this can be an issue thought.